diagnostic.md

```diff
@@ -31,39 +34,44 @@ the `form` DOM node. (Assume the list item has a class of "listerItem").
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+<!-- $("li").show -->
+
+<!-- $("li").()
+
+Setters
+$(.target).text("Hola")
```

Comment:
`.text()` is a setter method but trying to call `$(.target)` will cause an error.